pkg/virt-handler/migration.go

```diff
@@ -22,17 +22,26 @@ package virthandler
 import (
 	"fmt"
 	"os"
+	"time"
 
 	"gopkg.in/yaml.v2"
 )
 
 // FindMigrationIP looks for dedicated migration network migration0 using the downward API and, if found, sets migration IP to it
 func FindMigrationIP(networkStatusPath string, migrationIp string) (string, error) {
 	var networkStatus []NetworkStatus
+	var dat []byte
+	var err error
 
-	dat, err := os.ReadFile(networkStatusPath)
-	if err != nil {
-		return "", fmt.Errorf("failed to read network status from downwards API")
+	for i := 0; i < 4; i++ {
+		dat, err = os.ReadFile(networkStatusPath)
+		if err != nil {
+			return "", fmt.Errorf("failed to read network status from downwards API")
+		}
+		if len(dat) != 0 {
+			break
+		}
+		time.Sleep(500 * time.Millisecond)
```

Comment:
From your experience, are 1.5 seconds enough? Thanks

Reply:
Yes. 90% of the time, the file is there right away, and if not I've never seen it appear later than after the first sleep.
What I really want is to limit the delay on nodes that don't have the annotation, and therefore a file that will stay empty.
However, looking at this code again, there's a useless sleep on last iteration that I need to remove.
/hold

Reply:
Done, also added an extra loop to be extra cautious and match the PR/commit title. PTAL!
/hold cancel

Reply:
:sunglasses: